defaults-o2-full.sh

```diff
@@ -0,0 +1,63 @@
+package: defaults-o2-full
+version: v1
+env:
+  CXXFLAGS: "-fPIC -O2 -std=c++14"
+  CFLAGS: "-fPIC -O2"
+  CMAKE_BUILD_TYPE: "RELWITHDEBINFO"
```

Comment:
please set the `CXXSTD` environment variable as in defaults-o2.sh (just `-std=c++14` is sometimes not enough)

Reply:
Done! that was added after my copy of defaults-o2.sh I guess. Thanks for catching it.